compile.go

```diff
@@ -84,6 +86,10 @@ func CompileAST(ctx context.Context, astPkg *ast.Package, now time.Time, opts ..
 		return nil, err
 	}
 
+	if len(spec.Operations) == 0 {
+		return nil, errors.New("Flux script did not invoke any table-streaming functions.  Do you need to add a \"yield\" to your query?")
```

Comment:
This message is a little misleading as invoking a table streaming function is not enough. For example this produces an empty spec:

```
foo = from() |> ...
```

The assignment to `foo` means that the function calls did not produce a side effect, in order to get invoked.

Reply:
Yeah.  I'm struggling to find a concise way to express this in a friendly error message that tells the user how to fix the problem.  How about this?
```
this Flux script returns no streaming data.  Consider adding a "yield" or invoking streaming functions directly, without performing an assignment.
```

Reply:
@wolffcm That is much better. Thanks.